### PR TITLE
[ADD] mail_attach_image

### PR DESCRIPTION
**Pull request by ysantiago**
Based on the [res_users_signature](https://github.com/yelizariev/addons-yelizariev/tree/8.0/res_users_signature) by @yelizariev found in the following [thread](https://www.odoo.com/es_ES/forum/ayuda-1/question/how-to-insert-image-to-email-signature-24487)

We develop this module to resolve the issue when add an image in email template with its code base64, this was not displayed correctly in the email message.

skitch 45

With this new module, the image is displayed propertly:

email

### CODE REVIEW

File: mail_attach_image/README.rst [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+Credits
+=======
+
+
+Contributors
+------------
+
+
+* Ivan Yelizariev <@yelizariev>
```
Comment (yelizariev) [inline mail_attach_image/README.rst]: Kindly use my email address here: yelizariev@it-projects.info

File: mail_attach_image/__init__.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# coding: utf-8
+# © 2016 Vauxoo - http://www.vauxoo.com
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# info Vauxoo (info@vauxoo.com)
+# contributor: Ivan Yelizariev @yelizariev
```
Comment (moylop260) [inline mail_attach_image/__init__.py]: Ivan Yelizariev yelizariev@it-projects.info

File: mail_attach_image/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# © 2016 Vauxoo - http://www.vauxoo.com
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# info Vauxoo (info@vauxoo.com)
+# contributor: Ivan Yelizariev @yelizariev
```
Comment (moylop260) [inline mail_attach_image/__openerp__.py]: Ivan Yelizariev yelizariev@it-projects.info

**Comment by ivantodorovich** [comment]
I'm trying to test it, but it doesn't work for me.

I've created a template (Send Invoice by Email) with an image like this:

``` html
<p style="color: #70A032;">
<img style="margin-right: 20px;" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACgAAAAoCAYAAACM/rhtAAAC3ElEQVR42u3YzUsUYRzA8QwyTEPqFtgpoxeqg3WoDl4koiCChKToECbeEzpIHkP/gw4SHRKKCBKjoE4WbUFFFmQpmS+bL6u7uqv7Muu6O/Pre/gdfgxB5k6wwR4+7MPMjs+XfebZWdwkIiWtHFgOLAf+KbCj7+jfqMZuNOI82nAdHWjHBRzGdmwKwLoCt+EEOtGPlxjDHOJYRgEZOBjFc3ThYBCBu3AE9TpuRAfuYhBheBBlrSlBDKvIQpSDfjSjdqOBDbiJGUTMBH6eISiY14yOXQ0u+K5LIoQDxSxxPR7YKI3+iByiatlMnMUPOBqXUy5EWbNoKfYevIwUBHmIcjGMD0iboFGIHlvRY4vmFvAgahFzaC92k5zEvJnENbGrcOGYY/PoQxwTGME4IhCVg4OvSOBUsbv4GKJmOTPwTKA/+hnGNCSGkB73NCit8d8xjU+o2nCgatIwjYTdwXpOuRrn6XgJEX2NIw8HBfN3rhUVqC4hoRN4iGlI3ux4u7NzWICjcW8RxjJW9XwCgsGiA1Ub0mY5HRM1jHHfp+hgAH0QJHVpXyCORWQRxf6iA1UPRCVMYAo5EyhqBFOIKUEYT33vaw0kUN2BIA4H4gubMPdlFnlEzfJ6Ok5B1O0gA+swa5ZZzG4uYEwj0liC+LhIYsV8BYUCC1RnEIeYkDximNJz00j4vp4KEPUOP3V8L9BAdQ4LyJlfNEl80dBJDMHFvH0qaWjMPG0eBh6o6vDITO7CU3NKNMg1SzyDDFJYgOAJagINVJW4hQgmTciaRo8jq9FiNksW3diJXrPUgQZah9CDz2bDuGoNYUwjhTdoQYVeuxkhCJqCDvTbhxu4jwH04zF60akBv3v+NiCLIWz9V4FWhU5Ug2psWcc1zRC0+QNLyWtMoqpUA7sgaC3VwLMQvEdFKQbuRRwe9pRiYCW+QXAFO0oqUHVD8ApXSzGwFqdxEcfL/90qB5YD//fAX3iktTJoAShwAAAAAElFTkSuQmCC" />
<strong>Por favor, piense en el medio ambiente antes de imprimir este mensage.</strong>
</p>
```

When I send the message, the content strips the "src" of the image. Just like it did before.

``` html
<p style="color: #70A032;">
<img style="margin-right: 20px;" src="">
<strong>Por favor, piense en el medio ambiente antes de imprimir este mensage.</strong>
</p>
```

Is there something I'm not doing right?

**Pull request closed without merging**